### ISSUE TITLE
[EXT] Check if the model works

### ISSUE THREAD

**Issue by noisrucer** [noisrucer/sign-language-detection#22]
## Description

Run the model for at least 10 epochs to see if the mAP indeed increases for train/validation set.

**Comment by noisrucer** [comment]
DONE (low mAP)
